public/app/features/transformers/editors/ConcatenateTransformerEditor.tsx

```diff
@@ -57,29 +57,23 @@ export class ConcatenateTransformerEditor extends React.PureComponent<Concatenat
 
     return (
       <div>
-        <div className="gf-form-inline">
-          <div className="gf-form">
-            <div className="gf-form-label width-8">Name</div>
-            <Select
+        <InlineField label="Name" labelWidth={16} grow>
+          <Select
+            className="width-18"
```

Comment:
```suggestion
          <Select
            width={36}
```
This should be the same as the className's value.